scripts/spack/packages/raja/package.py

```diff
@@ -110,13 +120,27 @@ def initconfig_hardware_entries(self):
                     "CUDA_ARCH", 'sm_{0}'.format(cuda_arch[0])))
                 entries.append(cmake_cache_string(
                     "CMAKE_CUDA_ARCHITECTURES", '{0}'.format(cuda_arch[0])))
+
         else:
             entries.append(cmake_cache_option("ENABLE_CUDA", False))
 
         if '+rocm' in spec:
             entries.append(cmake_cache_option("ENABLE_HIP", True))
             entries.append(cmake_cache_path(
                 "HIP_ROOT_DIR", '{0}'.format(spec['hip'].prefix)))
+            # BEGIN AXOM EDIT
+            # Fix blt_hip getting HIP_CLANG_INCLUDE_PATH-NOTFOUND bad include directory
+            if self.spec.satisfies('%clang'):
+                hip_root = spec['hip'].prefix
+                rocm_root = hip_root + "/.."
+                clang_version= str(self.compiler.version)
+                hip_clang_include_path = rocm_root + "/llvm/lib/clang/" + clang_version + "/include"
+                entries.append(cmake_cache_path("HIP_CLANG_INCLUDE_PATH", hip_clang_include_path))
+
+            # C++ 14 error fix in camp
+            entries.append(cmake_cache_string("CMAKE_CXX_FLAGS","--std=c++14"))
```

Comment:
I fixed this in my branch by setting BLT_CXX_STD

Reply:
I tried compiling raja with only BLT_CXX_STD set, but run into C++14 errors when compiling on nevada.

Reply:
That is unfortunate.  Is it possible this is not working like advertised?

https://github.com/LLNL/blt/blob/49971b88e631aba76a9990c7702ceee47a7c3216/cmake/SetupCompilerStandards.cmake#L66-L72

Reply:
This needs to be guarded by at least compiler but preferably compiler and platform.  This will not work on XL or PGI for example...

Reply:
It is possible that camp is maybe not honoring `BLT_CXX_STD`